packages/@jsii/kernel/lib/api.ts

```diff
@@ -100,6 +100,19 @@ export interface LoadResponse {
   readonly types: number;
 }
 
+export interface InvokeScriptRequest {
+  readonly pkgname: string;
+  readonly script: string;
+  readonly args?: string[];
+}
+
+export interface InvokeScriptResponse {
+  readonly status: number | null;
+  readonly stdout: Buffer;
+  readonly stderr: Buffer;
+  readonly output: string[];
+}
```

Comment:
What is `output` for? Seems to be the same as `stdout`?

Also, I would make `status` be able to represent either a code or a signal, since these are the two ways a process can finish.

```suggestion
export interface InvokeScriptResponse {
  readonly status:
    | { code: cp.SpawnSyncReturns<unknown>['status'] }
    | { signal: cp.SpawnSyncReturns<unknown>['signal'] };
  readonly stdout: Buffer;
  readonly stderr: Buffer;
}
```